Documentation/Tools/DependencyWindow.md

```diff
@@ -1,9 +1,9 @@
 
 # Dependency Window
 
-Often in Unity it is difficult to gleam which assets are being used, and what is referencing them. The "Find References in Scene" option works great when you are only concerned with the current scene, but what about your entire Unity project? This is where the [Dependency Window](/Assets/MixedRealityToolkit/Inspectors) can be useful.
+Often in Unity it is difficult to gleam which assets are being used, and what is referencing them. The "Find References in Scene" option works great when you are only concerned with the current scene, but what about your entire Unity project? This is where the [Dependency Window](/Assets/MixedRealityToolkit.Tools/DependencyWindow) can be useful.
```

Comment:
>/Assets/MixedRealityToolkit.Tools/DependencyWindow

Looks like this was added before your PR, but I don't think a relative link like this resolves to anything in our generated docs.